core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -808,14 +807,11 @@ public void run() {
                                     .load(info.playable.getImageUri())
                                     .asBitmap()
                                     .diskCacheStrategy(ApGlideSettings.AP_DISK_CACHE_STRATEGY)
-                                    .into(-1, -1) // this resizing would not be exact, so we have
-                                                  // scale the bitmap ourselves
+                                    .centerCrop()
+                                    .into(iconSize, iconSize)
                                     .get();
-                            icon = Bitmap.createScaledBitmap(icon, iconSize, iconSize, true);
-                        } catch (InterruptedException e) {
-                            e.printStackTrace();
-                        } catch (ExecutionException e) {
-                            e.printStackTrace();
+                        } catch(Exception e) {
```

Comment:
If you were hoping to catch the OutOfMemoryError with this, it won't work. OutOfMemoryError doesn't inherit from Exception.

https://docs.oracle.com/javase/7/docs/api/java/lang/OutOfMemoryError.html